### ISSUE TITLE
Better handling of padded values

### ISSUE THREAD

**Issue by ivyleavedtoadflax** [wellcometrust/deep_reference_parser#31]
Padded values are currently handled in a custom way, but could probably be much better handled using tensorflow masking, see: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Embedding
